src/pages/LoginView.js

```diff
@@ -0,0 +1,27 @@
+import Login from "../components/Login";
+import { Link } from 'react-router-dom';
+import '../CSS/Login.css';
+
+const LoginView = () => {
+    return(
+        <div className="container">
+            <div className="login-div">
+                <h1>
+                    <b>Digital</b>Bank&nbsp;
+                    <svg xmlns="http://www.w3.org/2000/svg" width="16" height="16" fill="currentColor" className="bi bi-bank2" viewBox="0 0 16 16">
+                        <path d="M8.277.084a.5.5 0 0 0-.554 0l-7.5 5A.5.5 0 0 0 .5 6h1.875v7H1.5a.5.5 0 0 0 0 1h13a.5.5 0 1 0 0-1h-.875V6H15.5a.5.5 0 0 0 .277-.916l-7.5-5zM12.375 6v7h-1.25V6h1.25zm-2.5 0v7h-1.25V6h1.25zm-2.5 0v7h-1.25V6h1.25zm-2.5 0v7h-1.25V6h1.25zM8 4a1 1 0 1 1 0-2 1 1 0 0 1 0 2zM.5 15a.5.5 0 0 0 0 1h15a.5.5 0 1 0 0-1H.5z"/>
+                    </svg>
+                </h1> 
+                <Login/>
+                <div>
+                    Have an account?
```

Comment:
The "Don't" part was ok, you just needed to change the last part of the question.